packages/client/test/open-feature.spec.ts

```diff
@@ -39,13 +39,13 @@ describe('OpenFeature', () => {
     describe('Requirement 1.1.2.1', () => {
       it('should throw because the provider is not intended for the client', () => {
         const provider = mockProvider({ runsOn: 'server' });
-        expect(() => OpenFeature.setProvider(provider)).toThrowError(
-          "Provider 'mock-events-success' is intended for use on the server."
+        expect(() => OpenFeature.setProvider(provider)).toThrow(
```

Comment:
I changed this because `toThrowError` has been deprecated and I was already modifying this file.

Reply:
Was just about to ask. Thanks!